components/sidebar/sidebar_service.cc

```diff
@@ -335,7 +407,15 @@ std::vector<SidebarItem> SidebarService::GetHiddenDefaultSidebarItems() const {
           return default_item.built_in_item_type ==
                  added_item.built_in_item_type;
         });
-    default_items.erase(iter);
+    DCHECK(iter != default_items.end());
+    if (iter == default_items.end()) {
+      // This should not happen, since |LoadSidebarItems| ignores items which
+      // aren't included in |GetDefaultSidebarItems|.
+      LOG(ERROR) << "Bad built-in item loaded, cannot match to : "
+                 << static_cast<int>(added_item.type);
+    } else {
+      default_items.erase(iter);
+    }
```

Comment:
This is where the crash was. It shouldn't get here anymore because of the fix earlier in the stack, but it's better to keep a check here.